EpikCord/client/websocket_client.py

```diff
@@ -381,17 +381,40 @@ async def _voice_state_update(self, data: discord_typings.VoiceStateUpdateData):
         )  # TODO: Make this return something like (VoiceState, Member) or make VoiceState get Member from member_id
 
     async def _guild_delete(self, data: discord_typings.GuildDeleteData):
-        if guild := self.guilds.remove_from_cache(data["id"]):
+        if guild := self.guilds.remove_from_cache(int(data["id"])):
             await self.dispatch("guild_delete", guild)
 
     async def _interaction_create(self, data: discord_typings.InteractionCreateData):
         interaction = self.utils.interaction_from_type(data)
         await self.dispatch("interaction_create", interaction)
 
     async def _channel_create(self, data: discord_typings.ChannelCreateData):
+        """Event Fired when a channel is created
+
+            Dispatches ``channel`` to the defined event function(s)
+        """
         channel = self.utils.channel_from_type(data)
         self.channels.add_to_cache(channel.id, channel)
         await self.dispatch("channel_create", channel)
+    
+    async def _channel_update(self,data:discord_typings.ChannelUpdateData):
+        """Event Fired when a channel is modified/updated. This event dispatch the channel object before and after the modification.
```

Comment:
> This event dispatch the channel object before and after the modification.
Doesn't make sense
```diff
- This event dispatch the channel object before and after the modification.
+ This event dispatches the channel objects before and after the modification.
```
And not sure if it's guaranteed for the channel to be in cache, so maybe add a note.

Reply:
fixed in [102e178](https://github.com/EpikCord/EpikCord.py/pull/208/commits/102e17860a475f3f15493c2b8b06d05da0a83b9e)